packages/upstream/rpc.2.3.0/opam

```diff
@@ -27,6 +26,7 @@ depends: [
   "oasis" {build}
   "cppo"  {build}
   "async" {< "v0.10.0"}
+  "camlp4"
```

Comment:
Is it deliberate that these two differ and that we've added camlp4?

Reply:
No, it's a mistake upstream. I'll fix it in the next release